shells/fish/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup               cmake 1.1
 PortGroup               legacysupport 1.0
 
 github.setup            fish-shell fish-shell 3.2.2
-revision                0
+revision                1
```

Comment:
The below changes don't affect what the port installs, do they? They only affect the test phase? If so, the revision should not be increased.

Reply:
Yep, I misread Portfile reference